gammapy/spectrum/tests/test_flux_point_estimator.py

```diff
@@ -106,19 +109,22 @@ def test_str(fpe_pwl):
     def test_run_pwl(fpe_pwl):
         fp = fpe_pwl.run()
         actual = fp.table["norm"].data
-        assert_allclose(actual, [1.080933, 0.910776, 0.922278], rtol=1e-5)
+        assert_allclose(actual, [1.081434, 0.91077 , 0.922176], rtol=1e-5)
```

Comment:
Isn't a relative tolerance of 1e-5 too strict for minimization results?

Reply:
I agree `rtol=1e-2` should be sufficient.